models/intel/handwritten-english-recognition-0001/README.md

```diff
@@ -0,0 +1,52 @@
+# handwritten-english-recognition-0001
+
+## Use Case and High-Level Description
+
+This is a network for handwritten English text recognition scenario. It consists of a CNN followed by Bi-LSTM, reshape layer and a fully connected layer.
+The network is able to recognize English text consisting of characters in the [GNHK](https://goodnotes.com/gnhk/) dataset.
+
+## Example
+
+![](./assets/handwritten-english-recognition-0001.jpg) -> 'Picture ID. and Passport photo'
+
+## Specification
+
+| Metric                    | Value     |
+| ------------------------- | --------- |
+| GFlops                    | 1.3182    |
+| MParams                   | 0.1413    |
+| Accuracy on GNHK test subset (excluding images wider than 2000px after resized to height 96px with aspect ratio) | 81.5%     |
+| Source framework          | PyTorch\* |
+
+> **Note:** to achieve the accuracy, images from the GNHK test set should be preprocessed into single-line text images, using the coordinates from the accompanying JSON annotation files in the GNHK dataset, and then binarized using adaptive thresholding.
+
+## Accuracy Values
+
+This model adopts [label error rate](https://dl.acm.org/doi/abs/10.1145/1143844.1143891) as the metric for accuracy.
```

Comment:
probably it should be a comment to achived accuracy metric, no need in separated paragraph

Reply:
Updated.